src/main/java/org/broadinstitute/hellbender/utils/codecs/AbstractBCICodec.java

```diff
@@ -17,8 +17,10 @@
         implements FeatureOutputCodec<F, Writer<F>>, FeatureCodec<F, Reader<F>>, FeatureReaderFactory<F> {
 
     @Override
-    public Reader<F> getReader( final FeatureInput<F> input ) {
-        return new Reader<F>(input, this);
+    public Reader<F> getReader( final FeatureInput<F> input,
```

Comment:
thanks, I appreciate you including these even if they're not super helpful, just for parity.  A low priority future task is to wrap these in an options class of some sort so it's less hideous.